webrtc.html

```diff
@@ -7339,6 +7340,12 @@ <h2>Dictionary RTCRtpContributingSource Members</h2>
               127 is converted to 0, and all other values are converted using
               the equation: <code>10^(-rfc_level/20)</code>.</p>
             </dd>
+            <dt><dfn data-idl><code>rtpTimestamp</code></dfn> of type
+                <span class="idlMemberType">unsigned long</span></dt>
+            <dd>
+              <p>The RTP timestamp of the latest played out media that arrived
```

Comment:
This RTP timestamp will be from a remote clock, right? Do we need language here defining this similar to [remoteTimestamp](https://w3c.github.io/webrtc-stats/#dom-rtcremoteoutboundrtpstreamstats-remotetimestamp)?

Reply:
Let's reference [RFC3550] section 5.1

Reply:
Done!